backend/routes/XummApi.js

```diff
@@ -0,0 +1,382 @@
+import express from 'express'
+import xrpl from "xrpl";
+import fetch from 'node-fetch'
+import axios from 'axios'
+import XrplNFTHelper from './XrplNFTHelper.js';
+import XummSdk from 'xumm-sdk';
+
+
+const xummApi = ''
+const xummSecret = ''
+
+const pinataApiKey = ''
+const pinataSecretApiKey = ''
+
+const tempAddr = "rfRZeyG8YfSmKPqdX6PVLFJ5bdPCraDAsA"
+
+const minterAddress = "rhk5dvLzcQ9HywR4cTsms3TqunAb94bXmv"
+const minterKey = ""
+
+const sdk = new XummSdk.XummSdk(xummApi,xummSecret)
+
+const router = express.Router()
+
+
+
+
+
+
+const xummCall = async (payload) => {
+
+    const data =  await sdk.payload.create(payload)
+    return data
+}
+
+
+
+router.route('/mintTickets').post( (req,res) => { 

```

Comment:
Each route would need documentation on what params/ metadata is needed, because if this submission is to be accepted as the official version, full API reference need to be provided for event organizers.

Reply:
Comments have been added with proper param requirements as well as return results in xummapi.js. Examples of post / get requests with expected metadata body object can be found in /tests/app.test.js. I can add the contents of the comments in xummapi.js to the readme if needed.

Reply:
Adding those to the readme would be helpful I think. It can be hard to parse through a reference doc if you can't use ctrl + F over the contents.